plugins/qeta/src/utils/utils.ts

```diff
@@ -70,3 +70,19 @@ export const getFiltersWithDateRange = (filters: Filters) => {
   }
   return filters;
 };
+
+export const confirmNavigationIfEdited = (edited: boolean) => {{
+  const handleBeforeUnload = (event: BeforeUnloadEvent) => {
+    if (edited) {
+      event.preventDefault();
+      event.returnValue = true; // Included for legacy support, e.g. Chrome/Edge < 119
```

Comment:
If you return something here, doesn't it fix it for Chrome?

Reply:
Yes that did the thing! Strange :D Should maybe update mozzilla wiki with this info..